addons/medical_gui/functions/fnc_bloodLossToRGBA.sqf

```diff
@@ -22,15 +20,4 @@ params ["_bloodLoss"];
 private _frBL = 0 max (_bloodLoss / BLOOD_LOSS_RED_THRESHOLD) min 1;
 private _colorInt = ceil (_frBL * (BLOOD_LOSS_TOTAL_COLORS - 1));
 
-[
-    [1.00, 1.00, 1.00, 1], // #ffffff
-    [1.00, 0.95, 0.63, 1], // #fff1a1
-    [1.00, 0.88, 0.46, 1], // #ffe075
-    [1.00, 0.80, 0.33, 1], // #ffcb55
-    [1.00, 0.72, 0.24, 1], // #ffb73c
-    [1.00, 0.63, 0.15, 1], // #ffa127
-    [1.00, 0.53, 0.08, 1], // #ff8815
-    [1.00, 0.43, 0.02, 1], // #ff6d05
-    [1.00, 0.29, 0.00, 1], // #ff4b00
-    [1.00, 0.00, 0.00, 1]  // #ff0000
-] select _colorInt;
+missionNamespace getVariable format ["%1_%2", QGVAR(bloodLossColor), _colorInt]
```

Comment:
No newline at end of file.
I think we always do that?
so

```suggestion
missionNamespace getVariable format ["%1_%2", QGVAR(bloodLossColor), _colorInt]

```

Reply:
Correct.